backend/match-reporter/MatchReporter.ts

```diff
@@ -86,43 +94,46 @@ const createMatchResultMessage = (match, decorations) => {
   return messageParts.join(' ')
 }
 
-const getDecorationsForTeam = (team, decorations) => {
-  for (const decoration of decorations) {
-    if (team.every(player => player.name === decoration.player1 || player.name === decoration.player2)) {
-      return decoration
-    }
-  }
-  return {}
-}
-
-const createRankingChangeMessage = (oldRankings, newRankings) => {
-  const rankingChanges = oldRankings
-    .map((oldPlayer, index) => ({
-      name: oldPlayer.name,
-      oldRanking: index + 1,
-      newRanking: newRankings.findIndex(p => p.id === oldPlayer.id) + 1,
-    }))
-    .filter(ranking => ranking.oldRanking != ranking.newRanking)
+const ratingComparator = (a, b): number => b.rating - a.rating
 
-  if (rankingChanges.length === 0) {
-    return ''
-  }
-
-  return rankingChanges
-    .map(c => (
-      `${c.name} ${c.oldRanking}. ⟶ ${c.newRanking}.`)
-    )
-    .join('\n')
-}
-
-const hasLeaderboardChanged = (leaderboardSize, oldRankings, newRankings) => (
-  oldRankings.findIndex((oldPlayer, index) => oldPlayer.id !== newRankings[index].id) < leaderboardSize
-)
-
-const createPurposeMessage = async rankings => {
+const createPurposeMessage = async (rankings): Promise<string> => {
   const rankingsText = rankings
     .map((ranking, i) => `${i + 1}. ${ranking.name} (${ranking.rating})`)
     .join('\n')
   const purpose = 'TOP PLAYERS\n' + rankingsText
   return purpose
 }
+
+export default class MatchReporter {
```

Comment:
btw could you tun it into a named export?

Reply:
ok, sure! good idea

Reply:
Fixed - in https://github.com/salsita/foosball-rating/pull/103/commits/a62d62460c2477acf339cc077581a4520d1d5a29 and frontend errors in https://github.com/salsita/foosball-rating/pull/103/commits/cf98df2b8002545600f4dcefc45dc77deb957047

Reply:
I had an idea to put one command in root folder to lint both FE and BE but I had to move eslint dependencies to root package.json - made in https://github.com/salsita/foosball-rating/commit/87e3465bb1a304b7baf0ae876f5ae7e3c420e281 but not included to this PR yet. Maybe I would put it in a separate PR

Reply:
both variant are fine with me

Reply:
Great, I will make a separate PR for https://github.com/salsita/foosball-rating/commit/87e3465bb1a304b7baf0ae876f5ae7e3c420e281 then